packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -136,14 +135,17 @@ export class ProfilesCache {
      *
      * @returns {void}
      */
-    public registerCustomProfilesType(profileTypeName: string): void {
-        const exists = fs.existsSync(path.posix.join(`${os.homedir()}/.zowe/profiles/${profileTypeName}`));
-        if (!exists) {
-            throw new Error(
-                `Zowe Explorer Profiles Cache error: Tried to register a custom profile type named: ${profileTypeName} that does not yet exist. Extenders must call "zoweExplorerApi.getExplorerExtenderApi().initForZowe()" first.`
-            );
-        }
-        this.allExternalTypes.add(profileTypeName);
+    public registerCustomProfilesType(profileTypeName: string) {
```

Comment:
According to the `eslint` rules the function return type is required. I would leave the `void` type in place.